crates/swc_atoms/src/lib.rs

```diff
@@ -54,7 +54,6 @@ include!(concat!(env!("OUT_DIR"), "/js_word.rs"));
     feature = "rkyv",
     derive(rkyv::Archive, rkyv::Serialize, rkyv::Deserialize)
 )]
-#[cfg_attr(feature = "rkyv", archive_attr(repr(C), derive(bytecheck::CheckBytes)))]
 pub struct Atom(#[cfg_attr(feature = "rkyv", with(crate::EncodeAtom))] Arc<str>);
```

Comment:
I think we can remove `crate::EncodeAtom`?

Reply:
Nvm, we can do it later.

Reply:
If we're going to reintroduce bytecheck we'll still need it, is that correct? If that's the case I'd limit this revert in minimal, single scoped vhange so we can bring it back easily and also bisect easily later if needed.

Reply:
No it's workaround for https://github.com/rkyv/rkyv/issues/288